### PR TITLE
LPS-136416 Test fix SAML#AssertSSOWithEncryptionAndDecryption	

### PR DESCRIPTION
**Pull request by weller0246**
https://issues.liferay.com/browse/LPS-136416

### CODE REVIEW

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:sf

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
ci:test:saml:nocompile

**Comment by weller0246** [comment]
@gaborlovasdotcom  @ferenconodiD  please review my code and tell me your opinion before forwarding

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 1]
```diff
@@ -588,59 +588,78 @@ definition {
 		User.viewLoggedOutPG();
 	}
 
-	@description = "This is a use case for LPS-88924"
+	@description = "This is a use case for LPS-88924 using virtual instance"
 	@priority = "4"
 	test AssertSSOWithEncryptionAndDecryption {
 		property test.name.skip.portal.instance = "SAML#AssertSSOWithEncryptionAndDecryption";
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:595]: Please add property to limit your test case to use only a single tomcat instance via:
property app.server.bundles.size = "0";

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase [hunk 2]
```diff
@@ -588,59 +588,78 @@ definition {
 		User.viewLoggedOutPG();
 	}
 
-	@description = "This is a use case for LPS-88924"
+	@description = "This is a use case for LPS-88924 using virtual instance"
```
Comment (gaborlovasdotcom) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructureee/security/SAML.testcase:591]: Please use the suffix format as it is in other test cases: " - Using virtual instances" Example in L#1897

**Review by gaborlovasdotcom — CHANGES_REQUESTED** [review]
Please apply the changes in the comments.

**Comment by gaborlovasdotcom** [comment]
@weller0246 Ticket ID is missing from the PR's name.

**Review by gaborlovasdotcom — APPROVED** [review]
LGTM!

**Comment by gaborlovasdotcom** [comment]
ci:test:saml:nocompile

**Review by ferenconodiD — APPROVED** [review]
LGTM!

**Comment by gaborlovasdotcom** [comment]
ci:forward

**Pull request closed without merging**